geopandas/tests/test_extension_array.py

```diff
@@ -396,7 +396,7 @@ def test_unstack(self, data, index, obj):
 
             expected = obj_ser.unstack(level=level, fill_value=data.dtype.na_value)
             if obj == "series":
-                assert (expected.dtypes == object).all()
+                assert all(is_object_dtype(x) for x in expected.dtypes)
```

Comment:
Why this change?

Reply:
Ruff gave an E721 linting error on this: https://docs.astral.sh/ruff/rules/type-comparison/

When looking a bit into this, apparently checking the dtype is a bit of a fuss and can give different results depending on how you are doing it, so you need to be careful. Hence, using the function pandas offers to do this seemed like a good idea?

Reply:
Okay then.